src/main/java/com/ericsson/ei/waitlist/WaitListStorageHandler.java

```diff
@@ -82,8 +82,8 @@ public void addEventToWaitListIfNotExisting(String event, RulesObject rulesObjec
         }
     }
 
-    private String findEventInWaitList(JsonNode id) {
-        String condition = "{\"_id\" : \"" + id + "\"}";
+    private String findEventInWaitList(String id) {
+    	String condition = "{\"_id\" : \"" + id + "\"}";
```

Comment:
Why the change to 2 spaces here? Keep it the formatting the same in the entire file

Reply:
We have removed tabs, may be because of that it looks like this. will update.